api_tests/nodes/views/test_node_detail.py

```diff
@@ -296,6 +296,21 @@ def test_node_list_embed_identifier_link(self, app, user, project_public, url_pu
         link = res.json['data']['relationships']['identifiers']['links']['related']['href']
         assert '{}identifiers/'.format(url_public) in link
 
+    def test_node_shows_wiki_relationship_based_on_disabled_status_and_version(self, app, user, project_public, url_public):
+        url = url_public + '?version=latest'
+        res = app.get(url, auth=user.auth)
+        rel_link = res.json['data']['relationships'].get('wikis', False)
+        assert rel_link
+        project_public.delete_addon('wiki', auth=Auth(user))
+        project_public.save()
+        res = app.get(url, auth=user.auth)
+        rel_link = res.json['data']['relationships'].get('wikis', False)
+        assert rel_link is False
```

Comment:
Minor: It'd be slightly more correct and more concise to check

```python
assert 'wikis' in res.json['data']['relationships']
```

but not a big deal.